src/DataDefinitionsBundle/Form/Type/ExportProviderChoiceType.php

```diff
@@ -1,32 +1,22 @@
 <?php
-/**
```

Comment:
why'd you remove the copyright headers?

Reply:
I deleted `ProviderChoiceType` and created `ImportProviderChoiceType` without copyright, but git shows it like one file. It's already fixed